src/vehicle.cpp

```diff
@@ -3925,91 +3925,6 @@ void vehicle::operate_planter(){
     }
 }
 
-void vehicle::on_move(){
-    if(scoop_on){
-        operate_scoop();
-    }
-    if( planter_on ){
-        operate_planter();
-    }
-    if( plow_on ){
-        operate_plow();
-    }
-    if( reaper_on ){
-        operate_reaper();
-    }
-}
-void vehicle::operate_plow(){
```

Comment:
You should delete the other versions of the operate_* function. Those here are the good ones that work correctly.